backend/handler/database/roms_handler.py

```diff
@@ -44,10 +45,26 @@ def wrapper(*args, **kwargs):
 
 
 class DBRomsHandler(DBBaseHandler):
-    def _filter(self, data, platform_id: int | None, search_term: str):
+    def _filter(
+        self,
+        data,
+        platform_id: int | None,
+        collection_id: int | None,
+        search_term: str,
+        session: Session,
+    ):
         if platform_id:
             data = data.filter(Rom.platform_id == platform_id)
 
+        elif collection_id:
+            collection = (
+                session.query(Collection)
+                .filter(Collection.id == collection_id)
+                .one_or_none()
+            )
+            if collection:
+                data = data.filter(Rom.id.in_(collection.roms))
```

Comment:
so no way to filter by both platform and collection?

Reply:
fixed!